### DISCUSSION TITLE
Can't scrap reactors' profiles

### DISCUSSION THREAD

**Discussion by piotrsledz** [kevinzg/facebook-scraper#883] [Q&A] [answered]
Hello,
I wanted to write a script that would scrap all profiles that reacted on a post, i set
`options={"reactors": True}`
But reactors are still shown as "None"

The code below

```
from facebook_scraper import get_posts

for post in get_posts('xxx', pages=100,options={"reactors": True}):
    print(post[reactors])
```

**Comment by neon-ninja** [comment, marked as answer]
Try pass cookies as per the readme
